docs/adv-topics/autopatch.md

```diff
@@ -105,7 +119,7 @@ This table summarizes what they do:
 | dialtime | The maximum time to wait between DTMF digits when a telephone number is being dialed. The patch will automatically disconnect if this time is exceeded. The value is specified in milliseconds. |
 | farenddisconnect | When set to 1, the patch will automatically disconnect when the called party hangs up. The default is send a circuit busy tone until the radio user brings the patch down. |
 | noct | When this is set to 1, the courtesy tone during an autopatch call will be disabled. The default is to send the courtesy tone whenever the radio user unkeys. |
-| quiet | When set to 1, do not send dial tone, voice responses, just try to connect the call. |
+| quiet | When set to 1, do not send dial tone, voice responses, just try to connect the call. Set this to 1. |
```

Comment:
Not sure that I would have added the "Set this to 1." as a you really want to make this change.  Are there cases when one would want the variable set to 0 and other times 1?  and should we call those out?

Reply:
The call will fail if quiet is not set to 1, so I feel like it's a good idea to add that.

Reply:
I need to check to see what the default is for quiet.  I don't have that set on either of my systems and autopatch works and I get dial tone properly.

Reply:
I think the default is 0, because I had to set it to 1 to get autopatch working on my system using flowroute SIP.  Before setting it to 1, it was not specified at all, so I think the default is 0.

Reply:
Oh, I'm using a simplex node, so dialtone is not possible I believe.  Building a repeater over the next couple of days and can test full duplex autopatch.  I will remove the instruction to set it to 1 for now.

Reply:
I believe that dial tone worked on my simplex node.

Reply:
Interesting.  Just thinking about it and not testing, if the radio is transmitting the dial tone, how could it also receive the DTMF, if it's not full duplex?  I assumed that's why I had to set quiet=1 at the time I set it up.  But at this point I don't remember exactly what the error was without quiet=1.

Reply:
For a simplex patch, it transmits for a period of time, then pauses to see if the radio is receiving anything.  Vox is involved on the telephone side.  It works quite well.

Reply:
I thought I had already updated my commit and removed the mention to set it as 1, but I don't see it in the diff here and I don't have any changes to push.

Reply:
Looks like it's gone in the commit.